Controllers/Settings.php

```diff
@@ -36,8 +37,8 @@ public function init(SettingRepository $settingsRepo): void
      */
     public function get(): Response
     {
-        $projectCacheExpiration = (int) ($this->settingsRepo->getSetting('omnisearchsettings.projectscache') ?: 2400);
-        $ticketCacheExpiration = (int) ($this->settingsRepo->getSetting('omnisearchsettings.ticketscache') ?: 1200);
+        $projectCacheExpiration = (int) ($this->settingsRepo->getSetting('omnisearchsettings.projectscache') ?: 60);
```

Comment:
whyy

Reply:
2400 minutes turned about to be a bit much